platforms-serverless/serverless-framework-lambda/run.sh

```diff
@@ -3,9 +3,13 @@
 set -eux
 
 yarn install
-
 yarn prisma generate
-
 yarn tsc
 
+# only package
+yarn sls --version
+yarn sls package
```

Comment:
I this to check what would be packaged by `deploy` ?

Reply:
Yes exactly, this creates the package locally in .serverless _and leaves it there_ so we can upload it as an artifact later. I should add a few more comments here, so people do not use this as a template for their own CI.